pkg/domain/infra/abi/play.go

```diff
@@ -311,6 +313,8 @@ func (ic *ContainerEngine) PlayKube(ctx context.Context, body io.Reader, options
 			logrus.Infof("Kube kind %s not supported", kind)
 			continue
 		}
+		completedDocs++
+		fmt.Printf("Progress: %d / %d\n", completedDocs, totalDocs)
```

Comment:
Not sure but will this directly work for `podman-remote` use case since this will be only printed on the server side.

Reply:
Yes we should not print in backend CODE.